Hestia/app/src/main/java/hestia/backend/exceptions/ServerExceptions/UndefinedServerException.java

```diff
@@ -0,0 +1,14 @@
+package hestia.backend.exceptions.ServerExceptions;
+
+public class UndefinedServerException extends ServerException {
+    String message;
+        public UndefinedServerException(){
+            message= "InternalServerError";
+        }
+
+    @Override
+    public String getMessage() {
```

Comment:
Overriding the method might be a cause for strange behaviour. Better to change the name of the field.